docs/Lexicon.md

```diff
@@ -484,6 +484,39 @@ See [mandatory passes](#mandatory-passes--mandatory-optimizations).
 An implicit representation change that occurs when a value is used with
 a different [abstraction pattern](#abstraction-pattern) from its current representation.
 
+## realization
+
+The process of initializing an ObjC class for use by the ObjC runtime.
+This consists of allocating runtime tracking data, fixing up method lists
+and attaching categories.
+
+This is distinct from the initialization performed by `+initialize`,
+which happens only when the first message is sent to the class.
+
+The order of operations is: realization, followed by `+load` (if present),
+followed by `+initialize`. There are few cases where these can happen
+at different times.
+
+- Common case (no `+load` or special attributes): Realization is lazy and
+  happens when the first message is sent to a class. After that, `+initialize`
+  is run.
+- If the class has a `+load` method: `+load`, as the name suggests, runs at
+  load time; it is the ObjC equivalent of a static initializer in C++. For
+  such a class, realization eagerly happens at load time before `+load` runs.
+  (Fun aside: C++ static initializers run after `+load`.) `+initialize` still
+  runs lazily on the first message.
```

Comment:
This is a bit complicated because `+load` is itself a message, but it doesn't actually trigger `+initialize`. The first message other than `+load` triggers `+initialize`, which could be in the `+load` implementation, or if `+load` doesn't do any message sends, whenever the first message is sent elsewhere.

Not quite sure how to describe all that without a massive discussion. Maybe something like, "`+initialize` is not triggered by the `load` message itself` but runs lazily on the first other message sent."

Reply:
I added a "(other than `+load`)" when we say "first message" for the first time.

Reply:
That's much better than my awkward wording!